services/column-type-getter.js

```diff
@@ -57,6 +57,7 @@ function ColumnTypeGetter(databaseConnection, schema, allowWarning = true) {
 
     switch (type) {
       case 'BIT': // NOTICE: MSSQL type.
+      case 'BIT(1)': // NOTICE: MySQL type.
```

Comment:
[Spec] This was removed for a good reason, this doesn't works on postgresql 
![Capture d’écran 2019-10-31 à 14 10 19](https://user-images.githubusercontent.com/3263749/67949616-4891a600-fbe8-11e9-893c-ab6d7340ad6c.png)
I did not test on mysql but the argument is already enough to remove this line or find another way. By the way if I remember on some version of mysql (Because I worked on it 2 months ago I think) it was not working

Reply:
OK, thank you for testing this! 👍

> this doesn't works on postgresql

Good catch. Just for my understanding, what's the purpose of using `bit(1)` in postgreSQL since `boolean` exists? Anyway, I have to fix it!

> By the way if I remember on some version of mysql (Because I worked on it 2 months ago I think) it was not working

Could we find some history about this? Maybe it's fixed now: `sequelize` and `mysql2` were both upgraded.

Anyway, I have several options:
 - Revert my work about `bit(1)` (and open a new issue to fix this and test it with various MySQL version)
 - Consider it work with MySQL and modify the code to only consider this case for MySQL
 - something else.

What is your opinion @VincentMolinie ?

Reply:
I would prefer reverting the bit(1) and doing another PR for this issue as it is much harder/longer than the rest of the PR